packages/@ember/-internals/glimmer/tests/integration/components/input-angle-test.js

```diff
@@ -305,28 +312,33 @@ moduleFor(
       // this.assertAttr('tabindex', '30'); //NOTE: failing in IE (TEST_SUITE=sauce)
     }
 
-    ['@test dynamic attributes (named argument)']() {
-      this.render(
-        `
-      <Input @type="text" @value={{this.value}}
-        @disabled={{this.disabled}}
-        @placeholder={{this.placeholder}}
-        @name={{this.name}}
-        @maxlength={{this.maxlength}}
-        @minlength={{this.minlength}}
-        @size={{this.size}}
-        @tabindex={{this.tabindex}}
-      />`,
-        {
-          value: 'Original value',
-          disabled: false,
-          placeholder: 'Original placeholder',
-          name: 'original-name',
-          maxlength: 10,
-          minlength: 5,
-          size: 20,
-          tabindex: 30,
-        }
+    ['@test [DEPRECATED] dynamic attributes (named argument)']() {
+      maybeExpectDeprecation(
+        EMBER_MODERNIZED_BUILT_IN_COMPONENTS,
+        () => {
+          this.render(
+            `<Input @type="text" @value={{this.value}}
+            @disabled={{this.disabled}}
+            @placeholder={{this.placeholder}}
+            @name={{this.name}}
+            @maxlength={{this.maxlength}}
+            @minlength={{this.minlength}}
+            @size={{this.size}}
+            @tabindex={{this.tabindex}}
+          />`,
+            {
+              value: 'Original value',
+              disabled: false,
+              placeholder: 'Original placeholder',
+              name: 'original-name',
+              maxlength: 10,
+              minlength: 5,
+              size: 20,
+              tabindex: 30,
+            }
+          );
+        },
+        /Passing the `@.+` argument to <Input> is deprecated\./
```

Comment:
```suggestion
        /Passing the `@(disabled|placeholder|name|maxlength|minlength|size|tabindex)` argument to <Input> is deprecated\./
```

Let's scope this regex more narrowly. I realized we would want to make sure the deprecation is _not_ triggered for `@type` and `@value`.

Reply:
(same thing for the other `@.+` elsewhere)